transcom/mymove
# B 19724 add emplid to office queues

## Agility ticket
https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A945422

## INT PR
https://github.com/transcom/mymove/pull/13144

## Summary
Add the EMPLID column to all the office user queues and searches.  The filter should work for the column and also sorting.

### Setup test data
You will need to create several Coast Guard service members, or go into the DB and give some existing CG members an emplid.

### How to test

1. Login as a Service Counselor that has some moves available in the Counseling Queue from Coast Guard members.
2. Verify that I can see the new EMPLID column and it's populated.
3. Verify that the column sorts and that I can filter by the entire EMPLID and get the expected results back.
4. Navigate to the PPM Closeout Queue (Note, you will need to be logged in as a SC that has the USCG GBLOC).
5. Run through steps 2-3 above.
6. Navigate to the Move Search for SC, search for a move or customer, Run through steps 2-3 above.
8. Navigate to the Customer Search for SC, search for a customer, Run through steps 2-3 above.
9. Login as a TOO, run through steps 2-3 to verify that the column is as expected in the Task Order Queue.
10. As a TOO, navigate to the Search, search for a move or customer, run through steps 2-3 above.
11. Login as a TIO, run through steps 2-3 to verify that the column is as expected in the Payment Request Queue.
12. As a TIO, navigate to the Search, search for a move or customer, run through steps 2-3 above.
13. Login as a QAE user, search for a move or customer, run through steps 2-3 above.
